src/AoC2024/1/aoc24_1.h

```diff
@@ -0,0 +1,41 @@
+#ifndef AOC24_1_H
+#define AOC24_1_H
```

Comment:
there is `#pragma once` as alternative to the include guards. depending on your preferences choose on. both work.